tests/acceptance/visual-routes-test.js

```diff
@@ -0,0 +1,62 @@
+import { visit } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { percySnapshot } from 'ember-percy';
+import a11yAudit from 'ember-a11y-testing/test-support/audit';
+
+module('Acceptance | visual regression', function (hooks) {
+  setupApplicationTest(hooks);
+
+  // This is used to map the current **new** routes to what they used to be in the
+  // old ruby app. This is to allow us to compare what the new app looks like
+  // vs the old app.
+  const pages = [
+    { title: '/builds/', route: '/releases' },
```

Comment:
Would it make sense to add a code comment here explaining the `a11y` switch? E.g.
```suggestion
    { title: '/builds/', route: '/releases'/*, a11y: true */ },  /* set the a11y property of a page object to true to run the audit for this particular route */
```

Reply:
I like this suggestion to add clarity. thanks!